Src/CSharpier.Tests/FormattingTests/TestFiles/FileEncoding_UTF8.cst

```diff
@@ -2,3 +2,9 @@ public class ClassName
 {
     public void MethodName() { }
 }
+
+public enum MeetingLocation
+{
+    Café,
```

Comment:
Uh... do you want to make the test case `Café` like the bug? `CafÃ©` kind of makes no sense.

Reply:
I'm thinking whatever you are looking at this code with isn't interpreting the file correctly? This is what I see
![image](https://user-images.githubusercontent.com/3606529/212225766-ba23d839-7deb-4c91-8e28-48f06c5afedf.png)